ngraph/frontend/onnx_import/src/core/graph.cpp

```diff
@@ -66,6 +66,25 @@ namespace ngraph
         Graph::Graph(const ONNX_NAMESPACE::GraphProto& graph_proto, Model& model)
             : Graph(graph_proto, model, std::unique_ptr<GraphCache>(new GraphCache()))
         {
+            // Remove dangling Parameters
```

Comment:
I am rather fan of performing such transformation on model proto (something like `transform::fixup_legacy_operators`). @mateusztabaka What do you think of it?

If it is hard to implement, I think that it is better to move removing dangling parameters to end of constructor which takes `std::unique_ptr<GraphCache>&& cache`. In current version removing dangling params is not called for `Subgraph` constructor.

Reply:
> I am rather fan of performing such transformation on model proto (something like transform::fixup_legacy_operators). @mateusztabaka What do you think of it?

It could be done there, but I think we'd have to traverse whole graph and initializers. Doing it here is convenient, because we know the connections between the nodes and which of those are dangling.

> If it is hard to implement, I think that it is better to move removing dangling parameters to end of constructor which takes std::unique_ptr<GraphCache>&& cache. In current version removing dangling params is not called for Subgraph constructor.

I was thinking about this, but if we do this for Subgraph it breaks loop operator - e.g. some models don't use iteration_num, but I think we shouldn't remove it.

Reply:
Ok, it makes sense